### PR TITLE
Add note and example about RouterOS user rights + terminology fixes

### PR DESCRIPTION
**Pull request by soosp**
**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

## Checklist:

- [ ] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [ ] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/device_tracker.mikrotik.markdown [hunk 1]
```diff
@@ -30,37 +32,38 @@ Go to **IP** -> **Services** -> **API** and enable it.
...
   type: string
 port:
-  description: Mikrotik API port.
+  description: RouterOS API port.
   required: false
   default: 8728 (or 8729 if ssl is true)
   type: integer
 ssl:
-  description: Use api_ssl service instead of api.
+  description: Use api-ssl service instead of API.
```
Comment (soosp) [inline source/_components/device_tracker.mikrotik.markdown]: api and api-ssl are names of services in RouterOS in lowercase.
See: /ip service print

Comment (soosp) [reply source/_components/device_tracker.mikrotik.markdown]: There are the API for for developers, and there are api and api-ssl services.

File: source/_components/device_tracker.mikrotik.markdown [hunk 2]
```diff
@@ -70,17 +73,46 @@ method:
...
 /certificate add common-name="Self signed demo certificate for API" days-valid=3650 name="Self signed demo certificate for API" key-usage=digital-signature,key-encipherment,tls-server,key-cert-sign,crl-sign
 /certificate sign "Self signed demo certificate for API"
 /ip service set api-ssl certificate="Self signed demo certificate for API"
 /ip service enable api-ssl
 ```
-If everything is working you can disable the pure api service:
+
+Then add `ssl: true` to `mikrotik` device tracker entry in your `configuration.yaml` file.
+
+If everything is working fine you can disable the pure API service in RouterOS:
```
Comment (soosp) [inline source/_components/device_tracker.mikrotik.markdown]: It have to be lowercased.

File: source/_components/device_tracker.mikrotik.markdown [hunk 3]
```diff
@@ -70,17 +73,46 @@ method:
...
+To use this device tracker you need restricted privileges only. To enhance the security of your MikroTik device create a "read only" user who is able to connect via API only:
+
+```bash
+/user group add name=homeassistant policy=read,api,!local,!telnet,!ssh,!ftp,!reboot,!write,!policy,!test,!winbox,!password,!web,!sniff,!sensitive on,!dude,!tikapp
+/user add group=homeassistant name=homeassistant
+/user set password="YOUR_PASSWORD" homeassistant
+```
+
+Add the additional configuration to the `mikrotik` device tracker entry in your `configuration.yaml` file:
+
```
Comment (soosp) [inline source/_components/device_tracker.mikrotik.markdown:105]: Maybe it should be an another linkable_title.

**Review by soosp — COMMENTED** [review]
Thanks for the enhancements!

File: source/_components/device_tracker.mikrotik.markdown (hunk 1 above)
Comment (soosp) [reply source/_components/device_tracker.mikrotik.markdown]: API can be accessed via api or api-ssl services.

Comment (ludeeus) [reply source/_components/device_tracker.mikrotik.markdown]: How about "Use SSL to connect to the API."

Comment (soosp) [reply source/_components/device_tracker.mikrotik.markdown]: Good! I updated the PR.

**Review by frenck — APPROVED** [review]

**Comment by frenck** [comment]
Thank you @soosp! That looks great! 👍 
That deserves a medal 🥇

**Pull request merged**